### ISSUE TITLE
Cannot unzip powershell zipped folders

### ISSUE THREAD

**Issue by YanBeat** [thejoshwolfe/yauzl#88]
`Compress-Archive myfolderpath\* mycompressed.zip`

Unzipping with yauzl gives an error: 
```
Error: invalid characters in fileName: folder\file.js
    at ...\node_modules\yauzl\index.js:413:70
    at ...\node_modules\yauzl\index.js:618:5
    at ...\node_modules\fd-slicer\index.js:32:7
    at FSReqWrap.wrapper [as oncomplete] (fs.js:614:17)
```

**Comment by thejoshwolfe** [comment]
Sounds like same issue as https://github.com/thejoshwolfe/yauzl/issues/66#issuecomment-384835373 . can you confirm the version of .NET you're using?

**Comment by YanBeat** [comment]
4.5 - 4.6 win 10

**Comment by thejoshwolfe** [comment]
thanks! this i'll get a fix for this published shortly.

**Comment by thejoshwolfe** [comment]
published yauzl 2.10.0 which replaces backslashes by default. let me know if this solves your problem.
